### PR TITLE
To make vrouter compliant with controller

### PR DESCRIPTION
**Pull request by michal-clapinski**
Merging with Sagar's forks.

### CODE REVIEW

File: utils/SConscript [hunk 1]
```diff
@@ -27,8 +27,15 @@ for sdir in  subdirs:
 
 # Build libvrutil
 libvrutil = 'vrutil'
-libvrutil_objs = [env.Object('nl_util.lo', 'nl_util.c'), env.Object('udp_util.lo', 'udp_util.c'),
-                  env.Object('ini_parser.lo', 'ini_parser.c'), env.Object('vr_util.lo', 'vr_util.c')]
+
+if sys.platform.startswith('win'):
+    libvrutil_objs = [env.Object('nl_util.lo', 'nl_util.c'), env.Object('udp_util.lo', 'udp_util.c'),
```
Comment (sodar) [inline utils/SConscript]: Repetition of `nl_util.lo` and other objects is unnecessary. On Windows `libvrutils_objs` can be appended with `vr_win_util.lo`. What I mean, is something like this:

```python
libvrutil_objs = [ ... ]
if sys.playform.startswith('win'):
    libvrutil_objs.append(env.Object('vr_win_util.lo', 'vr_win_util.c'))
```

File: utils/vr_util.c [hunk 2]
```diff
@@ -593,7 +593,7 @@ vr_send_interface_delete(struct nl_client *cl, unsigned int router_id,
 
     memset(&req, 0, sizeof(req));
 
-    req.h_op = SANDESH_OP_DELETE;
+    req.h_op = SANDESH_OP_DEL;
```
Comment (sodar) [inline utils/vr_util.c]: Does sandesh generate code differently now? Could You elaborate?

File: utils/SConscript (hunk 1 above)
Comment (michal-clapinski) [inline utils/SConscript]: Thanks, my python is a little rusty

**Comment by michal-clapinski** [comment]
retest this please

**Pull request merged**